<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="Genome Analysis Blog">
    <meta property="og:title" content="
  Sequence Analysis - Genome Analysis Blog
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/beginnings/sequence-analysis/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/beginnings/sequence-analysis/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/beginnings/sequence-analysis/">
    <title>
  Sequence Analysis - Genome Analysis Blog
</title>

    <link rel="canonical" href="https://katharineme.com/beginnings/sequence-analysis/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="Genome Analysis Blog">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="Genome Analysis Blog" />
    

    <meta name="generator" content="Hugo 0.80.0" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">Genome Analysis Blog</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>Sequence Analysis</h1>
    </header>

    <p>The human genome is 3.3 billion base pairs long.</p>
<p>75% of RNA-Seq sequences will be ribosomal.</p>
<p>Replicates</p>
<ul>
<li>Technical replicate: sequence same sample 2x
<ul>
<li>NGS technical replicates are usually 97% matching or better</li>
</ul>
</li>
<li>Biological replicate: different individual sequenced under same conditions</li>
</ul>
<h2 id="reference-genomes">Reference Genomes</h2>
<ul>
<li>2 major verisons
<ul>
<li>GRCh
<ul>
<li>NIH Genome Reference Consortium</li>
<li>Latest release is GRCh38</li>
</ul>
</li>
<li>UCSC
<ul>
<li>Latest release is hg38</li>
<li>Previous release was hg19</li>
<li>After hg19 they decided to jump release number to match GRCh</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="decoy-genomes">Decoy Genomes</h2>
<p>When you sequence a human genome, you dont just get human DNA. If you sequence many human genomes, you begin to identify genomes of other organisms that live inside us. In order to conveniently identify and filter out these common non-human sequences, decoy genomes of these symbioitic viruses have been created that you can align to. Examples are:</p>
<ul>
<li>Epstein-Bar Virus
<ul>
<li>Causes mono</li>
<li>50% of people have it</li>
<li>Stays in white blood cells for a long time</li>
</ul>
</li>
<li>Cytomegalovirus
<ul>
<li>33% of people have it</li>
<li>Usually has no effect on health</li>
</ul>
</li>
</ul>
<p>Decoys can also be sets of human sequences that we currently don&rsquo;t know how to align. Some very long repetitive sequences are very hard to align especially when sequenced with short reads.</p>
<h2 id="general">General</h2>
<ul>
<li>Variant calling algorithms have a larger effect on what variants are “found” than alignment algorithms.</li>
<li>Novel variants, rare variants, and indels have more variance between pipelines than SNPs.</li>
<li>Genomes of European descent are better represented in the public genome knowledge base and are therefore variant called more accurately. Genomes of African descent (where there is greater genetic diversity) , because they are more different from the human reference and are therefore more likely to have “rare” variants, are prone to more inaccurate variant calling.</li>
</ul>
<h2 id="cool-papers">Cool Papers</h2>
<ul>
<li><a href="https://www.mdpi.com/2077-0383/9/1/132">Bioinformatics and Computational Tools for Next-Generation Sequencing Analysis in Clinical Genetics</a></li>
<li><a href="https://www.nature.com/articles/s41598-019-39108-2">Comparative analysis of whole-
genome sequencing pipelines to
minimize false negative findings</a></li>
</ul>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
