<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="The Art of Genome Analysis">
    <meta property="og:title" content="
  GRCH38 and Beyond - The Art of Genome Analysis
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/drafts/grch38_and_beyond/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/drafts/grch38_and_beyond/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/drafts/grch38_and_beyond/">
    <title>
  GRCH38 and Beyond - The Art of Genome Analysis
</title>

    <link rel="canonical" href="https://katharineme.com/drafts/grch38_and_beyond/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="The Art of Genome Analysis">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="The Art of Genome Analysis" />
    

    <meta name="generator" content="Hugo 0.115.3">
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">The Art of Genome Analysis</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>GRCH38 and Beyond</h1>
    </header>

    <p>From <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6699627/#SD1">this paper</a> on benchmarking practices:</p>
<p>&ldquo;Moving forward, groups will also need to modify benchmarking strategies to address changes in the way the human genome itself is represented. Today the most common way of representing the human genome involves a set of linear chromosomes (e.g., the most common usage of GRCh37). There are key advantages to non-linear representations of the genome, including ability to recognize copy-number and other polymorphisms directly in the reference, and as a result more graphical structures are in development.25 The GRCh38 build of the human genome makes a key step towards this with its use of ALT loci, which provide multiple distinct versions of specific regions of the genome.26 These ALT loci are not well-accounted for by most aligners or the benchmarking tools we describe, and their impact on benchmarking studies is largely unexplored and likely would require a variety of samples with differing ALT alleles. It is likely that the core representation of the genome will continue to evolve over time, and benchmarking tools will continue to evolve.&rdquo;</p>
<p>In this video we are going to talk about reference genomes. We&rsquo;ll cover the most popular reference genomes,how they were made, their limitations, and what changes we can expect in the future.</p>
<p>Why do we need a reference?
First, to review, the existence of a reference genome allows us to summarize a person by simply listing the 4 to 5 million places where they are different from the reference, instead of listing all 3.2 billion of their bases. Which is a massive convenience. However, since the reference is what everybody is comparing against, creating and maintaining the reference genome is a big responsibility.</p>
<p>History of the reference genome
still maintained by four organizations – the National Center for Biotechnology Information (NCBI), the Genome Institute at Washington University, the Wellcome Trust Sanger Institute, and the European Bioinformatics Institute – that were leading players in the original Human Genome Project. These organizations make up the GRC/ or genome reference consortium, which is responsible for keeping the reference genome at the forefront of accuracy and completeness, and is still plugging gaps and making refinements over a decade after the Human Genome Project was declared complete.</p>
<p>Most popular reference genomes, their names, whats equivalent to what, and how they are made.
WHo is in charge of creating and maintaining reference genomes?</p>
<p>NCBI36 from March 2006
grch37 released in march 2009
grch38 released in december 2013</p>
<p>grch37 and hg19 are the same</p>
<p>ghrch38 versus grch37</p>
<ul>
<li>Repair incorrect reads (Build 38 alters around 8,000 single nucleotides across the genome, which “in many cases will improve the annotation or analysis of clinically relevant genes,” )</li>
<li>inclusion of model centromere sequences</li>
<li>addition of alternate loci (Build 38 contains 261 alternate loci across 178 regions – one of which, the KIR locus involved in the differentiation of natural killer cells, has been given 35 distinct sequences.)</li>
<li>GRCh38 also adds new sequences to help fill those pieces of the genome that have never been fully captured</li>
</ul>
<p>This is the first reference to have centromere sequences. however there are still gaps present in the genome.</p>
<p>One novel resource that GRCh38 takes advantage of is a hydatidiform mole, a type of abnormal pregnancy that occurs when a sperm fertilizes an enucleated egg. That sperm then reduplicates its DNA, resulting in two identical copies of each chromosome. Since the resulting cell has no allelic variation, it can be used to generate unambiguous reads in regions of very high diversity.</p>
<p>The biggest leap in coverage in GRCh38 comes from the centromeres, which are modeled for the first time in the reference genome’s history. Once thought to be important only during cell division, centromeres have generated new interest as the evidence mounts that their sequences may have real impacts on function.</p>
<p>applied a new analytical approach to create estimated centromere sequences. Centromeres are defined largely by very slight variations on a massively repeated 171-base sequence</p>
<p>GRC did incorporate some changes to the MUC5AC gene based on a PacBio sequence posted in GenBank. As more sequences generated on long-read technology are placed in public repositories, the GRC will have more opportunity to benefit from advances in capturing these difficult regions.</p>
<p>CONVERSION TOOL from GRCH37 to GRCH38. MCBI remapping tool: <a href="https://www.ncbi.nlm.nih.gov/genome/tools/remap">https://www.ncbi.nlm.nih.gov/genome/tools/remap</a></p>
<p>Show a diagram of the life of reference genomes, as they progress from various sources.</p>
<p>Limitations of the latest and greatest reference. What regions we dont have sequence for, what was the sample pool for this genome.</p>
<h2 id="future">Future</h2>
<p>GRC says they:</p>
<p>&ldquo;have decided to indefinitely postpone our next coordinate-changing update (GRCh39) while we evaluate new models and sequence content from ongoing efforts to better represent the genetic diversity of the human pangenome, including those of the Telemore-to-Telomere Consortium and the Human Pangenome Reference Consortium&rdquo;</p>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
>
<p>SeqWell, CelSee Genesis, Becton Dickinson Rhapsody.</p>
<p>Between thousands and hundreds of thousands of microwells on a plate. The beads are engineered so yo get one bead in each well. Beads coat the microwell array. Then cells are applied. Small subset of wells will get a cell. Those wells will go through reverse transcription, amplification and library prep.</p>
<h2 id="analysis">Analysis</h2>
<h4 id="fastqc">FastQC</h4>
<h4 id="alignment">Alignment</h4>
<p>Tools such as Cell Ranger (commercial software from 10x Genomics) [Zheng et al., 2017], zUMIs [Parekh et al., 2018], alevin [Srivastava et al., 2019], RainDrop [Niebler et al., 2020], kallisto|bustools [Melsted et al., 2021], STARsolo [Kaminow et al., 2021] and alevin-fry [He et al., 2022] provide dedicated treatment for aligning scRNA-seq reads along with parsing of technical read content (CBs and UMIs), as well as methods for demultiplexing and UMI resolution.</p>
<p>STARsolo has performs very similarly to 10X genomics on 10x data, but runs ten times faster than CellRanger.</p>
<p><a href="https://kb.10xgenomics.com/hc/en-us/articles/115000794686-How-is-the-MEX-format-used-for-the-gene-barcode-matrices-">https://kb.10xgenomics.com/hc/en-us/articles/115000794686-How-is-the-MEX-format-used-for-the-gene-barcode-matrices-</a></p>
<p>Spliced mapping, contiguous mapping, and varieties of lightweight mapping.</p>
<p>Cell Barcode Assignment. This should be done by the aligner.</p>
<p>Get gene by cell matrix for each sample.</p>
<h4 id="clean-gene-by-cell-counts-matrix">Clean Gene by Cell Counts Matrix</h4>
<p>&ldquo;Cell QC is commonly performed based on three QC covariates: the number of counts per barcode (count depth), the number of genes per barcode, and the fraction of counts from mitochondrial genes per barcode (Ilicic et al, 2016; Griffiths et al, 2018). The distributions of these QC covariates are examined for outlier peaks that are filtered out by thresholding&rdquo;</p>
<p>from: <a href="https://www.embopress.org/doi/full/10.15252/msb.20188746">https://www.embopress.org/doi/full/10.15252/msb.20188746</a></p>
<p>Remove unvarying genes. Remove genes with all zeros.</p>
<p>Remove low quality cells. (emptyDrops via STARsolo flag)</p>
<p>Normalize sequencing depth. SCnorm, sctransform, and bayNorm.</p>
<p>From STARsolo manuscript:</p>
<p>&quot;
We further filtered the cells that contain &lt; 200 genes or &gt; 20% of mitochondrial
reads based on CellRanger counts, resulting in the final set of 4,473 cells which
was used for all tools.
2. For each tool, genes detected in less than 3 cells were excluded.
3. Counts were normalized to 10000 reads per cell, a pseudocount of 1 was added
and the natural log transformation was performed. These normalized expression
values were used for differential gene expression calculations (see below).
4. For UMAP embedding, neighborhood graph calculation and clustering, the ex- pression values were scaled to unit variance and zero mean across the cells and
truncated at a maximum value of 10.
5. 3, 000 highly variable genes were selected using the ’seurat v3’ algorithm.
&quot;</p>
<h4 id="heading">?</h4>
<p>Imputation and smoothing. SAVER. Fills some zeroes with imputed values based on overall structure of the data. Many false positives. Imposes patterns. Risky.</p>
<p>Cell cycle assignment. cyclone and Seurat. cyclone uses pairs of genes to asssign G1,S, G2/M, but doesn&rsquo;t recognize non-cycling cells well. Seurat scores cells based on expression of known cycle markers. Once cells have been assigned a stage, they both use linear model to regress out differences.</p>
<p>Feature selection. Select the genes with highest variance across cells, they have the most signal. Its been shown that noise follows a negative binomial distribution. Seurat selects features by empirically fitting the relationship between variance and mean expression. The reverse approach can also be taken, by removing features with more zeroes than expected.</p>
<p>Scale expression.</p>
<p>Reduce dimensions using PCA.</p>
<p>Determine significant principal components.</p>
<p>Use the principal components to cluster cells. Graph it.</p>
<p>Visualize clusters with non-linear dimensional reduction (UMAP, tSNE) using the principal components.</p>
<p>Detect and visualize marker genes for the clusters.</p>
<h2 id="sources">Sources</h2>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1.full.pdf">https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1.full.pdf</a></p>
<p><a href="https://www.nature.com/articles/s41596-020-00409-w">https://www.nature.com/articles/s41596-020-00409-w</a></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481139/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481139/</a></p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/28263961/">https://pubmed.ncbi.nlm.nih.gov/28263961/</a></p>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/28212749/">https://pubmed.ncbi.nlm.nih.gov/28212749/</a></p>
<p><a href="https://www.sc-best-practices.org">https://www.sc-best-practices.org</a></p>
<p><a href="https://hbctraining.github.io/Intro-to-rnaseq-hpc-salmon/lessons/04_quasi_alignment_salmon.html#:~:text=The%20quasi%2Dmapping%20approach%20estimates,base%2Dby%2Dbase%20alignment">https://hbctraining.github.io/Intro-to-rnaseq-hpc-salmon/lessons/04_quasi_alignment_salmon.html#:~:text=The%20quasi%2Dmapping%20approach%20estimates,base%2Dby%2Dbase%20alignment</a>.</p>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
