<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />

    <meta name="author" content="Katharine Medetgul-Ernar" />
    <meta name="description" content="https://katharineme.com/" />
    <meta name="keywords" content="blog,developer,personal" />

    <meta property="og:site_name" content="The Art of Genome Analysis" />
    <meta
      property="og:title"
      content="
  RNA-Seq Alignment - The Art of Genome Analysis
"
    />
    <meta property="og:description" content="" />
    <meta property="og:type" content="website" />
    <meta
      property="og:url"
      content="https://katharineme.com/posts/rna-seq-alignment/"
    />
    <meta property="og:image" content="https://katharineme.com/" />
    <meta name="twitter:card" content="summary" />
    <meta
      name="twitter:site"
      content="https://katharineme.com/posts/rna-seq-alignment/"
    />
    <meta name="twitter:image" content="https://katharineme.com/" />

    <base href="https://katharineme.com/posts/rna-seq-alignment/" />
    <title>RNA-Seq Alignment - The Art of Genome Analysis</title>

    <link
      rel="canonical"
      href="https://katharineme.com/posts/rna-seq-alignment/"
    />

    <link
      rel="stylesheet"
      href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700"
    />
    <link rel="stylesheet" href="/css/normalize.min.css" />
    <link rel="stylesheet" href="/css/style.min.css" />

    <link
      rel="icon"
      type="image/png"
      href="/images/favicon.png"
      sizes="32x32"
    />
    <link
      rel="icon"
      type="image/png"
      href="/images/favicon.png"
      sizes="16x16"
    />

    <link
      rel="alternate"
      href="https://katharineme.com/index.xml"
      type="application/rss+xml"
      title="The Art of Genome Analysis"
    />
    <link
      href="https://katharineme.com/index.xml"
      rel="feed"
      type="application/rss+xml"
      title="The Art of Genome Analysis"
    />

    <meta name="generator" content="Hugo 0.83.1" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
        <section class="container">
          <a class="navigation-title" href="/">The Art of Genome Analysis</a>
          <input type="checkbox" id="menu-control" />
          <label class="menu-mobile float-right" for="menu-control">
            <span class="btn-mobile float-right">&#9776;</span>
            <ul class="navigation-list">
              <li class="navigation-item align-center">
                <a class="navigation-link" href="https://katharineme.com/posts"
                  >Posts</a
                >
              </li>
            </ul>
          </label>
        </section>
      </nav>

      <div class="content">
        <section class="container post">
          <article>
            <header>
              <h1 class="title">RNA-Seq Alignment</h1>
              <h2 class="date">May 5, 2021</h2>
            </header>

            <p>
              Here we discuss how RNA-Seq alignment diverages from DNA-Seq
              alignment. But first, we need to fully understand RNA and RNA-Seq.
            </p>
            <h2 id="dna-vs-rna">DNA vs. RNA</h2>
            <p><img src="/images/dna_vs_rna.jpg" alt="sample depth" /></p>

            <p style="font-size: 18%; color: #8f8f8f; margin: 0">
              Photo credit to biologydictionary.net
            </p>

            <p>
              RNA and DNA are different molecules and hold different
              information. DNA is more or less fixed and consistent across all
              your cells, with the exception of de novo mutations (in skin cells
              from UV light for example), whereas RNA produced by the cell is
              changing all the time.
            </p>
            <p>
              From acetylation to methylation, alternative splicing, and other
              regulatory mechanisms, our cells use a bunch of different
              strategies to specialize the way they read and transcribe the
              genome, resulting in very different functions (why our skin cells
              are so different from heart cells).
            </p>
            <p>
              RNA is a bit like a snapshot of the cell. Results of RNA-Seq will
              be vastly different between cell types and over time in the same
              cell. For example, the cells might be fighting a pathogan or in
              the process of division, these will produce vastly different
              results. Even you may give off a very different impression if
              someone meets you at work versus out at a bar. Its the same
              concept. Cell type dependent and time dependent.
            </p>
            <p>
              In sum: DNA tells you what the cells are working with, RNA tells
              you what they&rsquo;re up to.
            </p>
            <h2 id="rna-seq">RNA-Seq</h2>
            <p>
              Its a bit of a misnomer because we don&rsquo;t actually sequence
              the RNA. We first convert it into cDNA, which is much more stable,
              then sequence it. But lets break it down step by step.
            </p>
            <h4 id="sampling">Sampling</h4>
            <p>What kind of RNA will be in the sample?</p>
            <ul>
              <li>85% is ribosomal RNA (rRNA)</li>
              <li>10-12% is transfer RNA (tRNA)</li>
              <li>2-3% is messenger RNA (mRNA)</li>
              <li>&lt;3% is lncRNA, miRNA, ncRNA, circRNA</li>
            </ul>
            <h4 id="library-preparation">Library Preparation</h4>
            <p><strong>1. Check for Degredation</strong></p>
            <p>
              Can be done with RNA gel. If mRNA is degraded, special treatments
              need to be done to repair transcripts first.
            </p>
            <p><strong>2. Enrich for mRNA</strong></p>
            <ul>
              <li>
                Poly-A Enrichment
                <ul>
                  <li>
                    Beads that bind to the poly A tails on mRNA trancripts and
                    pull them down, isolating them from other RNAs.
                  </li>
                  <li>
                    Cant be used in prokaryotic cells becuase they typically
                    dont have poly A tails on their mRNA transcripts.
                  </li>
                </ul>
              </li>
              <li>
                rRNA depletion
                <ul>
                  <li>Selectively remove rRNA sequences.</li>
                </ul>
              </li>
            </ul>
            <p><strong>3. Convert to cDNA</strong></p>
            <p>Why? Increases stability and its easier for sequencing.</p>
            <p><strong>4. Fragment</strong></p>
            <p>
              After conversion, fragments will be many different sizes, because
              transcripts vary in length. To fix this, we do fragmentation
              (enzymatic or via sonication). As usual, small biases in per base
              sequence content in the first ~10 bases can be introduced here.
            </p>
            <p>
              Agilent bioanalyzer can tell you how efficient fragmentation was
              and the distribution of fragement size in your sample.
            </p>
            <p><strong>5. Ligate Adapters</strong></p>
            <h4 id="sequencing">Sequencing</h4>
            <p>
              75 bp is a common length for RNA-Seq reads. Single end sequencing
              is more common in RNA-Seq than DNA-Seq.
            </p>
            <p>
              single end is sually enough for differential expression analysis.
              De novo sequencing or splice variant anlaysis would require paried
              end longer reads at greater depth.
            </p>
            <h2 id="alignment">Alignment</h2>
            <h2 id="normalization">Normalization</h2>
            <ol>
              <li>
                Normalize the number of reads per sample. If you didnt do this,
                you would think the gene expression of sample B is double the
                gene expression of sample C. Wherein reality, their gene
                expression maybe almost identical.
              </li>
            </ol>

            <img style="height: 230px" src="/images/sample_depth.png" />
            <p style="font-size: 18%; color: #8f8f8f; margin: 0">
              Photo credit to Biorender
            </p>

            <ol>
              <li>Normalize by gene length.</li>
            </ol>
            <h2 id="interpretation">Interpretation</h2>
          </article>

          <br />
        </section>
      </div>
    </main>

    <script src="/js/app.js"></script>
  </body>
</html>
