<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="Genome Analysis Blog">
    <meta property="og:title" content="
  Tabix - Genome Analysis Blog
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/posts/tabix/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/posts/tabix/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/posts/tabix/">
    <title>
  Tabix - Genome Analysis Blog
</title>

    <link rel="canonical" href="https://katharineme.com/posts/tabix/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="Genome Analysis Blog">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="Genome Analysis Blog" />
    

    <meta name="generator" content="Hugo 0.80.0" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">Genome Analysis Blog</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container post">
  <article>
    <header>
      <h1 class="title">Tabix</h1>
      <h2 class="date">January 13, 2021</h2>

      
    </header>

    <p>If tabix were a character, it would be the swordsman Roronoa Zoro from One Piece, slicing through your genome data to deliver you the result you want.</p>
<p><img src="/images/zoro.jpeg" alt="command line"></p>
<p>Tabix is part of Samtools. It indexes a tab-delimted genome position file (VCF file for example). Once indexed, it can quickly retrieve data from any part of the file without decompressing it. Thank you Heng Li üôè (tabix author). Here are some things you can do with tabix to a <code>vcf.gz</code> file.</p>
<p>First, you&rsquo;ll want to index your VCF like this.</p>
<p><img src="/images/tabix_make_index.png" alt="command line"></p>
<p>The index file is called <code>.tbi</code> and will be put adjacent to your <code>.gz</code> file.</p>
<p><img src="/images/tabix_index_made.png" alt="command line"></p>
<p>Now you can check for the existence of a variant like so.</p>
<p><img src="/images/tabix_one_variant.png" alt="command line"></p>
<p>Its important to remember, while you&rsquo;re doing this, that a polymorphism being in the VCF file is not synonymous with being interesting or important or rare. There are lots of harmful / interesting / important polymorphisms in the reference genome. But more on that in another post.</p>
<p>You can look for variants in a region in the same way.</p>
<p><img src="/images/tabix_region.png" alt="command line"></p>
<p>You can also do other useful / cool things with tabix, like return the header of a VCF.</p>
<div class="highlight"><pre class="chroma"><code class="language-sh" data-lang="sh">tabix -H vcf.gz
</code></pre></div><p>Return chromosome names of a VCF.</p>
<div class="highlight"><pre class="chroma"><code class="language-sh" data-lang="sh">tabix -l vcf.gz
</code></pre></div><p>You can also return variants found in regions listed in a file. The file can be a bed (.bed, .bed.gz, .bed.bgz) or a TAB-delimited file with CHROM, POS, and, optionally, POS_TO columns, where positions are 1-based and inclusive.</p>
<p>I like to use BED files. BED files have only 3 required columns: chromosome, start position, and end position. They have 9 more optional columns. The only optional column I use is the 4th one, which is <code>name</code>. I use it because it helps me remember what that locus is. More on the BED file format <a href="https://m.ensembl.org/info/website/upload/bed.html">here</a>. Here is a BED file I might use.</p>
<div class="highlight"><pre class="chroma"><code class="language-fallback" data-lang="fallback">chr7    150999023    150999023   rs1799983
chr19   51354484     51354484    rs79338777
chr21   36146408     36146408    rs1056892
</code></pre></div><p>So to search for variants in the locations listed in that BED file, we enter:</p>
<div class="highlight"><pre class="chroma"><code class="language-sh" data-lang="sh">tabix -R test.bed vcf.gz
</code></pre></div><h2 id="careful-with-that-gz-file-">Careful with that .gz file &hellip;</h2>
<p><code>gzip</code> and <code>bgzip</code> are not the same thing. Wait &hellip; what is <code>bgzip</code>? Good question. <code>bgzip</code> is a compression algorithm that creates a series of <code>gzip</code> compressed blocks which are each 64kb in size. To go along with all these little blocks is whats called a &ldquo;virtual offset&rdquo;. The vitual offset is actually an unsigned (meaning nonnegative) 64 bit integer. And this integer holds information which acts like a map of the <code>bgzip</code> file. Its because of the &ldquo;block + map&rdquo; structure that <code>bgzip</code> files can be accessed without unzipping the entire file. And this is why tabix can access lets say <code>chr1:452923-452923</code> without unzipping the whole <code>variants.vcf.gz</code> (a bgzipped file).</p>
<p>But why did I say be careful? Because <code>bgzip</code> files and <code>gzip</code> files both have the filename extension <code>.gz</code>. So how do you know if a <code>.gz</code> file is bgzipped or just gzipped? Because for the reasons explained above, you cannot access a part of a <code>gzip</code> file without unzipping it, which is why like tabix requires <code>bgzip</code> compression. Luckily its easy to tell. Just try to tabix it (create an index) and if its not bgzipped, tabix will tell you:</p>
<p><img src="/images/tabix_not_bgzipped.png" alt="command line"></p>
<p>Learn more about <code>bgzip</code> and <code>gzip</code> in <a href="https://blastedbio.blogspot.com/2011/11/bgzf-blocked-bigger-better-gzip.html">Peter Cock&rsquo;s detailed post</a>.</p>

  </article>

  <br/>
  
</section>

      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
