<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="The Art of Genome Analysis">
    <meta property="og:title" content="
  Types of DNA Tests - The Art of Genome Analysis
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/posts/types_of_dna_tests/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/posts/types_of_dna_tests/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/posts/types_of_dna_tests/">
    <title>
  Types of DNA Tests - The Art of Genome Analysis
</title>

    <link rel="canonical" href="https://katharineme.com/posts/types_of_dna_tests/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="The Art of Genome Analysis">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="The Art of Genome Analysis" />
    

    <meta name="generator" content="Hugo 0.90.0" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">The Art of Genome Analysis</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container post">
  <article>
    <header>
      <h1 class="title">Types of DNA Tests</h1>
      <h2 class="date">January 28, 2021</h2>

      
    </header>

    <p>In this video I&rsquo;m going to compare three methods for checking a persons DNA: whole genome sequencing, whole exome sequencing, and genotyping arrays.</p>
<p>Each of these methods are common types of consumer DNA tests, but they are hugely different in terms of usability.</p>
<p>First to review, the human genome is 3.2 billion base pairs long. 1.2% of the genome is genes, and the rest is DNA that sits between gene sequences, called intragenic DNA. Genes are the sequences that are transcribed into proteins. And proteins are the workhorses of the cell. Which is why if there&rsquo;s a problem in a gene sequence, there will likely be a problem with the protein that gene makes. That protein may not do its job correctly, which sometimes gives the carrier a sign or symptom.</p>
<p>So you might think, ah, then only the sequences of genes matter, we can just focus on those. Which would be great, but the genome is more complex than this. Nature didn&rsquo;t just stick all that intragenic DNA in our genomes for fun. Many intragenic DNA sequencs help regulate how and when specific genes are transcribed. Other intragenic regions are crucial for how DNA physically folds, which in turn affects how genes are read. And we are learning more everyday about the many roles intragenic DNA plays.</p>
<p>This brings us to the primary way these three genomic tests differ: their coverage.</p>
<p>Coverage</p>
<p>Whole genome sequencing delivers the entire 3.2 billion base pairs of an individual. Whole exome sequencing delivers only the gene sequences of an individual (so about 1.2% of the genome or about 38 million base pairs). And genotyping delivers information on a few hundred thousand basae pairs which is about .01% of the genome.</p>
<p>Cost</p>
<p>Now the genome coverage (or how much of the genome sequence they deliver) affects the prize. As you can imaging, wgs is the most expensive at about 1000 per genome for clinical grade quality for 99% accuracy. Whole exome sequencing on the otehr hand is about $400 and genotyping is anywhere from $50 to $200.</p>
<p>How they work</p>
<p>In terms of how these test actually work. Whole genome sequencing and whole exome sequencing are quite similar. First DNA is isolated from the sample, then it is broken into little pieces. In wgs, all these DNA fragments get little ends called adapters stuck to them so that they can stick to the flow cell in the sequencing machine. However in wes, the exons, or regions that code for genes, are isolated from the sea of DNA fragments and then have adapters attached to them for sequencing. Thereafter the sequencing process is nearly identical.</p>
<p>However genotyping works very differently. Instead of sequencing the DNA base by base to discover the sequence, genotyping starts with a device called an array. You can think of an array like a sea of sticky genomic flags. Each flag will only bind to a specific sequence. So when the sample DNA is broken into tiny pieces and washed over the array, there will be signal where the array DNA sequence matched and bound to the sample sequence.</p>
<p>Now what are these DNA sequences on the array lawn? Mostly they are the sequences of common mutations seen in humans. So the type of data you get when you genotype a sample is a series of yes and nos to all to all the mutated sequences that the array has on its lawn. For example, you might get four thousand yeses and 400 thousand nos. Which really means that in the end, these four thousand sequences are the only things you know exist in the sample DNA.</p>
<p>But when we start to think about all the mutations the sample DNA has that weren&rsquo;t on the array lawn, it kind of makes you sweat. Considering that we know of about 350 million human variants, and that an array can only test about 600K variants at most, its clear that the amount of information an array yeilds is very miniaml.</p>
<p>Usability</p>
<p>At the end of the day, we usually do a DNA test to get some value from it. We have some questions about this persons DNA we want answered. So naturally these tests vary in how effectively they can address these questions. First lets say you want to know where your ancestors are from.</p>
<p>In this case, the more of your DNA sequence we have, the more accurately we can tell you what your ancestry is. So whole genome sequencing would be the best. Here is why. Ancestry is a very complex. Stay tuned for another video where I go in depth into the challenges and strategies for determining genetic ancestry. A population of people doesn&rsquo;t differ from another by having a mutation that the other population doesnt have. This almost never happens. Instead two populations differ by how common a variation is in their population. For example, variant X might occur in 20% of population A, but 60% of population B. Therefore if you have variant X, you are slightly more likely to come from population B than population A. Therefore, the more variants like this that can be considered to determine ancesty, the more statically powerful the ancestry prediction becomes. This is why, genotyping for determining ancestry can be inaccurate, because only a small number of your variants are being considered.</p>
<p>Now if you are interested in finding out whether you carry a specifc mutation, then a genotyping array would work just fine as long as it includes the mutation you are interested in. The catch would be that, lets say this mutation is related to breast cancer. New mutations important for cancer are discovered everyday. And so by the time you get your genotyping results, its possible that the array didnt check for the 20 other mutations that are consequential for breast cancer. Leavig you to believe that you have  your answer when really many important checks were not even done.</p>



<p style="text-align:center;">
    <img src="/images/dance.jpeg" style="height:450px; padding: 10px;">
</p>

<p>compare on</p>
<p>coverage</p>
<p>usability</p>
<ul>
<li>disease risk</li>
<li>ancestry</li>
</ul>
<p>acurracy</p>
<p>future of the data</p>
<p>Even though the genome is a sequence of nucleotides A, T, G, and C, its more of an ecosystem than a book.</p>
<p>23andme for example uses a version of the Illumina Global Screening Array.</p>

  </article>

  <br/>
  
</section>

      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
