<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="The Art of Genome Analysis">
    <meta property="og:title" content="
  SNP Microarray - The Art of Genome Analysis
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/drafts/snp_microarray/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/drafts/snp_microarray/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/drafts/snp_microarray/">
    <title>
  SNP Microarray - The Art of Genome Analysis
</title>

    <link rel="canonical" href="https://katharineme.com/drafts/snp_microarray/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="The Art of Genome Analysis">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="The Art of Genome Analysis" />
    

    <meta name="generator" content="Hugo 0.90.0" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">The Art of Genome Analysis</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>SNP Microarray</h1>
    </header>

    <p>From <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6699627/#SD1">this paper</a> on benchmarking practices:</p>
<p>&ldquo;Moving forward, groups will also need to modify benchmarking strategies to address changes in the way the human genome itself is represented. Today the most common way of representing the human genome involves a set of linear chromosomes (e.g., the most common usage of GRCh37). There are key advantages to non-linear representations of the genome, including ability to recognize copy-number and other polymorphisms directly in the reference, and as a result more graphical structures are in development.25 The GRCh38 build of the human genome makes a key step towards this with its use of ALT loci, which provide multiple distinct versions of specific regions of the genome.26 These ALT loci are not well-accounted for by most aligners or the benchmarking tools we describe, and their impact on benchmarking studies is largely unexplored and likely would require a variety of samples with differing ALT alleles. It is likely that the core representation of the genome will continue to evolve over time, and benchmarking tools will continue to evolve.&rdquo;</p>
<p>Most popular reference genomes, their names, whats equivalent to what, and how they are made.</p>
<p>Show a diagram of the life of reference genomes, as they progress from various sources.</p>
<p>Limitations of the latest and greatest reference. What regions we dont have sequence for, what was the sample pool for this genome.</p>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
er the microarray and adenine is added. The laser excites the label on the adenine and a yellow signal is produced. Remember, even though I&rsquo;m only showing one oligonucleotide probe here , there are actually hundreds of this exact probe on this bead. So you can imagine 100s of instances of this reaction occurring simultaneously, producing a strong yellow signal.</p>
<p>The second possibility is that the sample is homozygous for G at this SNP. In that case, once the sample&rsquo;s complement DNA leading up to SNP location is bound, a G is added. After being excited by the laser, a blue signal is produced, which again is amplified by this same reaction happening hundreds of times across the surface of this bead.</p>
<p>The third possibility is that the sample is heterozygous and has a genotype of AG. In this case, some of the hundreds of oligonucleotide probes would had an adenine added, and some would have a Guanine added. This would happen of course because the sample DNA has roughly equal amounts of DNA fragments complementing each, in this case roughly equal amounts of T and C at this location in the sample genome. Therefore when the laser excites the nucleotide labels, half of the signal will be yellow and half will be blue, resulting in a green signal, indicating the heterozygous genotype AG.</p>
<p>And this is happening simultaneously on each of the hundreds of thousands of beads on the SNP microarray. In the end, the allele at each locus is known with some level of confidence. That level of confidence varies depending on the strength and clarity of the light signal.</p>
<p>When it comes down to it, performing a microarray assay is a bit like dna fishing. It works great for summarizing a individual or sample in the space of known variants.</p>
<p>If you still have questions comment below and I will actually respond. If this was helpful, please like and subscribe. Have a healthy happy day everyone.</p>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
