<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="The Art of Genome Analysis">
    <meta property="og:title" content="
  Read Pre-Processing - The Art of Genome Analysis
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/posts/read-preprocessing/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/posts/read-preprocessing/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/posts/read-preprocessing/">
    <title>
  Read Pre-Processing - The Art of Genome Analysis
</title>

    <link rel="canonical" href="https://katharineme.com/posts/read-preprocessing/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="The Art of Genome Analysis">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="The Art of Genome Analysis" />
    

    <meta name="generator" content="Hugo 0.88.1" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">The Art of Genome Analysis</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container post">
  <article>
    <header>
      <h1 class="title">Read Pre-Processing</h1>
      <h2 class="date">September 15, 2020</h2>

      
    </header>

    <p>Pre-processing is an umbrella term that can mean filtering out / trimming low quality reads and reads contaminated with adapter sequences, and / or correcting for sequencing baises such as batch or lane effects.</p>
<h2 id="sequencing-biases">Sequencing Biases</h2>
<ul>
<li><strong>Batch effects</strong> include any errors that occur after random fragmentation of the DNA until it is input to the flow cell.
<ul>
<li>e.g., PCR amplification and reverse transcription artifacts.</li>
</ul>
</li>
<li><strong>Lane effects</strong> include any errors that occur from the point at which the sample is input to the flow cell until data are output from the sequencing machine.
<ul>
<li>e.g., systematically bad sequencing cycles and errors in base calling.</li>
</ul>
</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881125/">Source</a></li>
</ul>
<h2 id="ways-to-trim-reads">Ways to Trim Reads</h2>
<ul>
<li>Hard Trim
<ul>
<li>Cut off reads after certain number of bases</li>
<li>Computational easy</li>
<li>Makes all reads same length</li>
</ul>
</li>
<li>Soft Trim
<ul>
<li>Trimming that considers quality scores in a sliding window that is usually ~10bp wide</li>
<li>Tools that do this
<ul>
<li>Sickle from UC Davis Bioinformatics</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="sequence-duplicaiton">Sequence Duplicaiton</h2>
<ul>
<li>Duplicates should be remeved</li>
<li>There are two ways duplication can happen:
<ul>
<li>PCR duplication from uneven amplification</li>
<li>Optical duplication
<ul>
<li>One sequencing colony can look like two colonies to the sequencing machine</li>
<li>Illumina made this less of an issue on the NovaSeq by adding millions of microwells to the flowcells</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="adapter-trimming">Adapter trimming</h2>
<ul>
<li>Adapters are almost always trimmed from 3’ end instead of 5’ end. Illumina reads don’t need to have the 5’ end adapter trimmed because there is a primer sequence that binds to the 5’ end and sequencing by synthesis begins with the first base on the DNA insert.</li>
<li>Most adapter trimmers will take into account the quiality of the sequences on the 3' end to better determine whether there is adapter sequence or not</li>
<li>It&rsquo;s good practice to run FASTQC before and after trimming for comparison</li>
<li><a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/illumina-adapter-sequences-1000000002694-14.pdf">Illumina adapter sequences</a></li>
</ul>
<h2 id="most-popular-pre-processors">Most popular pre-processors</h2>
<h3 id="skewer">Skewer</h3>
<ul>
<li>2015</li>
<li>Adapter trimmer</li>
<li>Less popular, but cited in good papers indicating quality</li>
<li>Not maintained, git issues are more or less ignored</li>
<li>Poor documentation</li>
</ul>
<h3 id="trimmomatic">Trimmomatic</h3>
<ul>
<li>2014</li>
<li>Adapter trimmer for Illumina reads</li>
<li>Fairly documented</li>
<li>Not actively maintained by main developer</li>
<li>Somewhat maintained by other unrelated contributors</li>
<li>Popular, praised in the community</li>
</ul>
<h3 id="cutadapt">Cutadapt</h3>
<ul>
<li>2011</li>
<li>Adapter trimmer</li>
<li>Well documented</li>
<li>Actively maintained</li>
<li>Some doubts about quality in the community</li>
</ul>
<h3 id="fastp">Fastp</h3>
<ul>
<li>2018</li>
<li><a href="https://github.com/OpenGene/fastp">https://github.com/OpenGene/fastp</a></li>
<li>All around pre-processor, many utilities</li>
<li>Very fast, supposedly 2x-5x faster than other pre-processors / trimmers</li>
<li>Well documented</li>
<li>Actively maintained. 800 stars on Github, &gt;100 issues, &gt; 100 issues resolved</li>
<li>Hot right now</li>
</ul>
<p>You can find at least one source for each that argues it is the best adapter / pre-processing tool. I have highest hopes for fastp.</p>
<h2 id="run-fastp">Run fastp</h2>
<p>Paramters to use for paired end short read sequencing.</p>
<ul>
<li>&ndash;adapter_sequence</li>
<li>&ndash;adapter_sequence_r2</li>
<li>—-in1</li>
<li>—-in2</li>
<li>—-out1</li>
<li>—-out2</li>
<li>—-unpaired1</li>
<li>—-unpaired2</li>
</ul>

  </article>

  <br/>
  
</section>

      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
