<!DOCTYPE html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <meta name="author" content="Katharine Medetgul-Ernar">
    <meta name="description" content="https://katharineme.com/">
    <meta name="keywords" content="blog,developer,personal">

    <meta property="og:site_name" content="The Art of Genome Analysis">
    <meta property="og:title" content="
  Trace Ancestry Through DNA - The Art of Genome Analysis
">
    <meta property="og:description" content="">
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://katharineme.com/beginnings/trace-ancestry-through-dna/">
    <meta property="og:image" content="https://katharineme.com/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="https://katharineme.com/beginnings/trace-ancestry-through-dna/">
    <meta name="twitter:image" content="https://katharineme.com/">

    <base href="https://katharineme.com/beginnings/trace-ancestry-through-dna/">
    <title>
  Trace Ancestry Through DNA - The Art of Genome Analysis
</title>

    <link rel="canonical" href="https://katharineme.com/beginnings/trace-ancestry-through-dna/">
    
    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Mono:400,700">
    <link rel="stylesheet" href="/css/normalize.min.css">
    <link rel="stylesheet" href="/css/style.min.css">

    

    

    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="32x32">
    <link rel="icon" type="image/png" href="/images/favicon.png" sizes="16x16">

    
      <link rel="alternate" href="https://katharineme.com/index.xml" type="application/rss+xml" title="The Art of Genome Analysis">
      <link href="https://katharineme.com/index.xml" rel="feed" type="application/rss+xml" title="The Art of Genome Analysis" />
    

    <meta name="generator" content="Hugo 0.88.1" />
  </head>

  <body class="">
    <main class="wrapper">
      <nav class="navigation">
  <section class="container">
    <a class="navigation-title" href="/">The Art of Genome Analysis</a>
    <input type="checkbox" id="menu-control"/>
    <label class="menu-mobile  float-right " for="menu-control">
      <span class="btn-mobile  float-right ">&#9776;</span>
      <ul class="navigation-list">
        
          
            <li class="navigation-item  align-center ">
              <a class="navigation-link" href="https://katharineme.com/posts">Posts</a>
            </li>
          
        
        
      </ul>
    </label>
  </section>
</nav>


      <div class="content">
        
  <section class="container page">
  <article>
    <header>
      <h1>Trace Ancestry Through DNA</h1>
    </header>

    <p>Ever since the human genome was sequenced in 2000, we have been trying to use it to figure out where we came from.</p>
<p>However, this field is at the intersection of society-wide misunderstandings about race / ethnicity, all the problems that come with a new technology (genome sequencing), and a public shortage of sequencing data for diverse human genomes. The obstacles between us and the &ldquo;truth&rdquo; about our genetic heritage are currenlty great. But these obstacles will be slowly toppled by better technology, better bioinformatics, and more human genome sequences.</p>
<h2 id="up-to-now">Up to Now</h2>
<p>So where are we now?</p>
<h2 id="comprehensive-analysis-of-genetic-ancestry-and-its-molecular-correlates-in-cancer-paper-2020">Comprehensive Analysis of Genetic Ancestry and Its Molecular Correlates in Cancer Paper (2020)</h2>
<p>They determined genetic ancestry using 5 different &ldquo;methods&rdquo;.</p>
<p>Among the  9,257 patients for whom at least three methods provided base calls, 98.1 percent exhibited agreement and 99.7 percent of admixed patients exhibited condordance with prior ancestry assignments from Yuan et al 2018.</p>
<p>The data they used for ancestry classification was either SNP array and or whole-exome sequencing.</p>
<h2 id="integrated-analysis-of-genetic-ancestry-and-genomic-alterations-across-cancers-2018">Integrated Analysis of Genetic Ancestry and Genomic Alterations across Cancers (2018)</h2>
<p>They aimed to estimate the genetic ancestry of each TCGA patient, including the 12% of TCGA samples that have no race / ethniticty classification, by applying multiple already made computational tools</p>
<p>EIGENSTRAT (Price et al., 2006) was applied to infer the global genetic structure of TCGA patients.</p>
<p>is a method to study human diversity based on Principal Component Analysis (PCA), which reduces the information contained in SNP frequencies to components that capture most genetic variability. The EIGENSOFT package (EIGENSTRAT algorithms included) version 6.1.4 was downloaded from GitHub (<a href="https://github.com/DReichLab/EIG)">https://github.com/DReichLab/EIG)</a>.</p>
<p>The smartpca program was applied to run PCA on the combined genotype data of TCGA and the reference populations.</p>
<p>we adopted the default value K=10 for running the smartpca program.</p>
<p>Overall, the number of patients classified into the four genetic ancestry groups (EA, AA, EAA and NA) remained stable after randomly changing the number of snps fed into the program, suggesting that a small number of SNPs is sufficient to assess global ancestry</p>
<p>STRUCTURE (Pritchard et al., 2000) was used to quantitatively determine the ancestral composition for each patient</p>
<p>is an algorithm using multi-locus genotype data to infer population structure utilizing a model-based clustering method. The STRUCTURE algorithm (version 2.3.4) was downloaded from <a href="http://web.stanford.edu/group/pritchardlab/structure.html">http://web.stanford.edu/group/pritchardlab/structure.html</a>. The USEPOPINFO model was applied in our analysis.</p>
<p>To estimate the genetic ancestry at a particular chromosomal locus, LAMP (Sankararaman et al., 2008) was applied to the genotype data of TCGA.</p>
<p>There was a 95.6% overlap between TCGA patient written race and identified global ancestry.</p>
<h2 id="principal-components-analysis-corrects-for-stratification-in-genome-wide-association-studies">Principal components analysis corrects for stratification in genome-wide association studies</h2>
<h2 id="genetic-ancestry-analysis-on-93000-individuals-undergoing-expanded-carrier-screening-reveals-limitations-of-ethnicity-based-medical-guidelines">Genetic ancestry analysis on &gt;93,000 individuals undergoing expanded carrier screening reveals limitations of ethnicity-based medical guidelines</h2>
<p><a href="https://www.nature.com/articles/s41436-020-0869-3">https://www.nature.com/articles/s41436-020-0869-3</a></p>
<p>Self-reported ethnicity was an imperfect indicator of genetic ancestry, with 9% of individuals having &gt;50% genetic ancestry from a lineage inconsistent with self-reported ethnicity.</p>
<p>39.6% of individuals cannot identify the ancestry of all four grandparents,</p>
<p>Current professional guidelines by the American College of Medical Genetics and Genomics (ACMG) and the American College of Obstetricians and Gynecologists (ACOG) recommend pan-ethnic carrier screening for cystic fibrosis and spinal muscular atrophy.2,3,4 In addition, these two professional societies have long recommended carrier screening for a partially overlapping and expanded set of conditions based on a patient’s self-reported ethnicity (SRE)</p>
<p>We analyzed the genetic ancestry of 93,419 individuals undergoing routine ECS for over 90 serious Mendelian disorders via NGS.</p>
<p>Genetic ancestry was inferred using a proprietary analysis method (patent application WO2018144135A1). In brief, the method is based on sparse non-negative matrix factorization (with seven components) and was used to estimate ancestry shared with hypothetical ancestral populations using an imputed reference panel of 51 populations and an additional imputed reference panel of the Ashkenazi Jewish population.</p>
<p>Overall, relative to pan-ethnic ECS, ethnicity-based guidelines would have identified only 23% of carriers in the study cohort, with large differences per ethnicity (Fig. 4, SI Table 4). Just over two-thirds (68%) of carriers among African Americans would have been identified based on screening guidelines from ACMG and ACOG; that number decreases to only 13% of carriers identified among Hispanics when guidelines are followed.</p>
<h2 id="inferring-genetic-ancestry-opportunities-challenges-and-implications">Inferring Genetic Ancestry: Opportunities, Challenges, and Implications</h2>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0002929710001552">https://www.sciencedirect.com/science/article/pii/S0002929710001552</a></p>
<p>Its an older paper from 2010, but its a great overview of the state of genetic ancestry analysis.</p>
<p>Each part of the genome has its own ancestry, we are made up of dna from lots of people.</p>
<p>Continental / race ancestry</p>
<p>Biogeographic ancestry</p>
<ul>
<li>a persons origin is associated with the geographic locations of presumed ancestors</li>
</ul>
<p>we cannot recreate out ancestors genomes, we dont know what they are. but assuming that the people who live in a given geographical area are a good represention of people who lived there a long time ago, is a rather large assumption.</p>
<p>also, just because you lived somewhere, or came from somewhere, doesnt mean you are genetically similar to the people who have historically lived there.</p>
<p>If we are being honest, linking &ldquo;ancestry&rdquo; to a physical location on earth is not as useful as simply understanding your genome in the context of all other genomes. What &ldquo;group&rdquo; or component you belong to, doenst necessarily corerlate to a physical place where people have lived.</p>
<p>There is probably a relationship between location and your genome at this point, however future generations may slowly loose this association.</p>
<p>AIM - an ancestry informative marker.</p>
<p>Currently, we only have partial knowledge of how human genetic diversity is distributed across the globe, but initial studies38 are revealing the degree of resolution possible in testing the relationship between genetic ancestry and geographic origins. A number of these studies have used a collection of ∼1100 DNA samples obtained from 51 populations living in different parts of the world; these samples constitute the Human Genetic Diversity Panel (HGDP).39 Analysis of 987 microsatellites typed in the HGDP collection, for example, inferred six population clusters that correspond to continental regions (i.e., Africa, America, Central/South Asia, East Asia, and Oceania)</p>
<h2 id="up-next">Up Next</h2>
<p>How will we improve?</p>
<h2 id="sources">Sources</h2>
<p><a href="https://www.sciencedirect.com/science/article/pii/S1535610820302117?casa_token=zp7T4IwBxIQAAAAA:botdxT6ZyGsA2HdUgT-3W9s_LWgO2lkigHgrvEj1pqmL3VeDaE4IRdERi5v2AZC0WykmfpqM-EQ">https://www.sciencedirect.com/science/article/pii/S1535610820302117?casa_token=zp7T4IwBxIQAAAAA:botdxT6ZyGsA2HdUgT-3W9s_LWgO2lkigHgrvEj1pqmL3VeDaE4IRdERi5v2AZC0WykmfpqM-EQ</a></p>
<p>Self-reported ethnicity was an imperfect indicator of genetic ancestry, with 9% of individuals having &gt;50% genetic ancestry from a lineage inconsistent with self-reported ethnicity. Limitations of self-reported ethnicity led to missed carriers in at-risk populations: for 10 ECS conditions, patients with intermediate genetic ancestry backgrounds—who did not self-report the associated ethnicity—had significantly elevated carrier risk.</p>

  </article>
</section>


      </div>
      
    </main>

    

  <script src="/js/app.js"></script>
  
  </body>
</html>
